install_config/web_console_customization.adoc

```diff
@@ -896,3 +895,28 @@ xref:../install/index.adoc#install-planning[inventory file]:
 # See: https://docs.openshift.com/enterprise/latest/install_config/aggregate_logging.html
 #openshift_master_logging_public_url=https://kibana.example.com
 ----
+
+[[changing-the-web-console-url-port-and-certificates]]
+== Changing the Web Console URL Port and Certificates
+
+To ensure your custom certificate is served when hitting the web console URL,
+add the certificate and URL to the *_master-config.yaml_* file under
+`NamedCertificates` on the master hosts. See
+xref:../install_config/certificate_customization.adoc#configuring-custom-certificates[Configuring
+Custom Certificates for the Web Console or CLI] for more information.
+
+To add a new URL under `redirectURIs` to set up redirects, change the
+`openshift-web-console oauthclient` with:
+
+----
+$ oc edit oauthclient openshift-web-console
+----
+
+To ensure users are correctly redirected, update the `PublicUrls` for the
+`openshift-web-console configmap`:
+
+----
+$ oc edit configmap/webconsole-config -n openshift-web-console
+----
+
+Update the values for `consolePublicURL` and `masterPublicURL`.
```

Comment:
It's unclear to me if you're changing both master public URL or just console URL. There are probably more changes needed if changing master public URL. If you haven't changed master URL, the `masterPublicURL` property in the config map doesn't need to be updated.

Do we have any existing topics that cover changing master public URL?

Reply:
@spadgett I don't see much mentioned about changing the master public URL. It's only mentioned briefly in places like https://docs.openshift.com/container-platform/3.9/install_config/master_node_configuration.html#master-config-oath-authentication-config and https://docs.openshift.com/container-platform/3.9/install_config/master_node_configuration.html#master-config-asset-config

Reply:
@spadgett @deads2k  Any more thoughts?

Reply:
@spadgett @deads2k Bump

Reply:
@deads2k Okay, thanks! This is updated.